Alphabetize the contributors list

jsumners opened the pull request:
This PR organizes the contributors list into alphabetical order by surname.

@cagataycali does this list yours in the correct position? I must admit, I do not know how to sort the `Ç` with respect to the English alphabet.

mcollina commented on package.json:
```diff
@@ -31,25 +31,25 @@
     {
-      "name": "Tomas Della Vedova",
-      "url": "http://delved.org"
-    },
```
Tomas was completely removed.

jsumners replied:
Not so. It is second in the list. Although, I have learned that the name should probably be alphabetized by "Vedova" instead of "Della".

delvedor replied:
Nope, my surname is "Della Vedova" :P

jsumners commented:
The list is alphabetized by surname because that's how you alphabetize names in English. See the section on names in https://simple.wikipedia.org/wiki/Alphabetical_order

lmammino commented:
Thanks for the clarification @jsumners. I'll take care of updating the sorting logic in the website once this is done.

jsumners commented:
BTW, I wish I could have found a better citation for the rule, but evidently I don't know how to search for that information.

lmammino commented:
Also, should we keep the `package.json` in sync with the actual list of contributor displayed on the website and on the README?

jsumners commented:
@lmammino yes, but I don't have the information to fill in the gap there.

mcollina commented on package.json:
```diff
@@ -31,25 +31,29 @@
+      "name": "Tommaso Allevi",
+      "email": "tomallevi@gmail.com"
+    },
+    {
```
I would prefer for Tomas to be the 1st of the contributors, if you are ok with it.

allevo replied:
The wrong Tom :smile:

jsumners replied:
I get it. I wish package.json supported more than a single author. It definitely needs more support for a large project developed by multiple people.

delvedor commented:
Is there anything else we can do here?

jsumners commented:
@delvedor other than re-position your entry in `package.json`, not that I'm aware of.

jsumners commented:
I have moved @delvedor back to the top of the contributors list in `package.json`. I also added a `"author": true` property to his entry to convey why its not in alphabetical order.

delvedor commented on .vscode/launch.json:
```diff
@@ -0,0 +1,14 @@
+      "program": "${workspaceFolder}/test/handler-context.test.js"
+    }
+  ]
+}
```
Is this file needed?

jsumners replied:
How did that get in there?

jsumners replied:
Fixed.

cagataycali commented:
@jsumners, you right, ‘Ç’ isn’t English. I’ll replace to ‘C’

delvedor commented:
Can we merge this?

jsumners commented:
I think so. But I was waiting on a LGTM or two.
